src/eko/strong_coupling.py

```diff
@@ -67,20 +68,55 @@ def as_expanded(order, as_ref, nf, scale_from, scale_to):
     res = as_LO
     # NLO
     if order >= 1:
-        beta1 = beta(1, nf)
-        b1 = beta1 / beta0
+        b1 = beta_b(1, nf)
         as_NLO = as_LO * (1 - b1 * as_LO * np.log(den))
         res = as_NLO
         # NNLO
-        if order == 2:
-            beta2 = beta(2, nf)
-            b2 = beta2 / beta0
+        if order >= 2:
+            b2 = beta_b(2, nf)
             res = as_LO * (
                 1.0
                 + as_LO * (as_LO - as_ref) * (b2 - b1**2)
                 + as_NLO * b1 * np.log(as_NLO / as_ref)
             )
-
+            # N3LO expansion is taken from Luca Rottoli and it's simpler
+            # than the APFEL expansion which is more accurate
```

Comment:
Can you quantify the level of discrepancy with APFEL?  And what's the difference that makes you call it "more accurate"?
(I expect something like "the expansion order", that if it's ridiculously large I agree we can keep this one)

Reply:
This was actually a @felixhekhorn guess, but I think the comment can be dropped...

Reply:
The comment is a bit misleading ...
-  the N3LO is coming from Luca and I don't believe there is such a thing in Apfel
- the "more accurate" should refer to NNLO - since we take that from Apfel and not from Luca
- than "more accurate" is what has been discussed maybe around ~June 20 (searching in the minutes): APFEL does not implement the ordinary expansion (as done by Luca), but an advanced expression that a) we don't have an explanation for and b) solves the RGE better, i.e. the subleading terms are numerically smaller (this is also discussed in one of my off-line MMa noteboooks)

Reply:
Ok, fine. So, no discussion on N3LO, and we don't need to discuss NNLO here (that in any case is something we've already done).

Reply:
@giacomomagni or @felixhekhorn, remove or improve the comment, please

Reply:
I'll drop it!